prettier/index.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
cool! will we want to make this part of our normal `mixmax/*` eslint configurations at some point?

Reply:
Possibly, yeah.

Reply:
Depends on your philosophy on composable eslint configs

Reply:
True, true. I figure given that we're adopting prettier across the board that we'll end up always composing them together, which means it's a good candidate for DRYing.